### ISSUE TITLE
Loader Utils Deprecation Warning - webpack

### ISSUE THREAD

**Issue by hugofqueiros** [vuejs/vue-cli#370]
Got this while running 'npm run dev':

(node:8702) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.

webpack as a bundler,
node v6.9.4
npm 3.10.10

**Comment by git-lt** [comment]
I got same error

```
> node build/dev-server.js

(node:87121) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.


 ERROR  Failed to compile with 3 errors                                                                                09:56:05

These dependencies were not found in node_modules:

* strip-ansi in ./~/.npminstall/webpack-hot-middleware/2.17.0/webpack-hot-middleware/client.js?noInfo=true&reload=true
* ansi-html in ./~/.npminstall/webpack-hot-middleware/2.17.0/webpack-hot-middleware/client-overlay.js
* html-entities in ./~/.npminstall/webpack-hot-middleware/2.17.0/webpack-hot-middleware/client-overlay.js

Did you forget to run npm install --save for them?
> Listening at http://localhost:8080
```
node v6.7.0
npm 3.10.3

**Comment by posva** [comment]
Fixed with vuejs/vue-loader#671
Update the vue-loader to get the fixed version
